control/vehicle_cmd_gate/package.xml

```diff
@@ -4,9 +4,11 @@
   <name>vehicle_cmd_gate</name>
   <version>0.1.0</version>
   <description>The vehicle_cmd_gate package</description>
-  <maintainer email="h_ohta@ertl.jp">h_ohta</maintainer>
+  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
   <license>Apache License 2.0</license>
 
+  <author email="h_ohta@ertl.jp">h_ohta</author>
```

Comment:
To be the same as the following line. cc @h-ohta 
https://github.com/autowarefoundation/autoware.universe/blob/a8a58c045e718d6e3193e9521adf2ba9eaad7ce2/common/tier4_state_rviz_plugin/package.xml#L7


```suggestion
  <author email="hiroki.ota@tier4.jp">Hiroki OTA</author>
```